deps/openssl/README.md

```diff
@@ -53,7 +53,7 @@ Here is a list of supported architectures for use of ASM in OpenSSL.
   | linux     | arm64      | linux-aarch64        | o   |
   | linux     | ppc        | linux-ppc            | o   |
   | linux     | ppc64      | linux-ppc64          | o   |
-  | linux     | ppc64      | linux-ppc64le        | o   | (node_byteoder: little)
+  | linux     | ppc64      | linux-ppc64le        | o   |(*1)
```

Comment:
It is stiil not in the table:

https://github.com/shigeki/node/blob/cb6aee5a3d8201fb656a8e971b3741d85976a9d0/deps/openssl/README.md#supported-architectures-for-use-of-asm

Maybe it should be places in a cell or its own column should be added?

Reply:
Sorry, I missed to view and check markdown. Fixed to include it in the cell as f8cdc0f.